opentimelineio/opentime.py

```diff
@@ -524,6 +542,28 @@ def to_frames(time_obj, fps=None):
     return int(time_obj.value_rescaled_to(fps))
```

Comment:
Could you add a `int()` around the `return time_obj.value` above also? That would make these two code branches consistent.